local/install_deps_linux.bash

```diff
@@ -152,6 +152,20 @@ pip install --upgrade pip
 pip install --upgrade -r docker/ci/requirements.txt
 pip install --upgrade -r src/local/requirements.txt
 
+# Install the Android emulator and its dependencies. Used in tests and as an
+# option during Android test case reproduction.
```

Comment:
Use local ANDROID_SDK_INSTALL_DIR=local/bin/android-sdk
Also version 28 and 4333796 should be local vars that can be easily tweaked ANDROID_SDK_VERSION=28 and ANDROID_SDK_REVISION=4333796